phoenix-core/src/it/java/org/apache/phoenix/end2end/AppendOnlySchemaIT.java

```diff
@@ -112,9 +114,9 @@ private void testTableWithSameSchema(boolean notExists, boolean sameClient) thro
             }
             
             // verify getTable rpcs
-            verify(connectionQueryServices, sameClient ? never() : times(1))
-                    .getTable((PName) isNull(), eq(new byte[0]),
-                            eq(Bytes.toBytes(viewName)), anyLong(), anyLong());
+            verify(connectionQueryServices, never())
```

Comment:
Is this change required?

Reply:
This test was checking if there was a `CQSI.getTable` call when a different connection is used to execute the same create ddl. With the new design, there will be no call to `CQSI.getTable` since the other connection will find the table metadata in the CQSI cache.

Reply:
Got it